### ISSUE TITLE
Why requesting for node.js 14.0 ?

### ISSUE THREAD

**Issue by vladoro** [microlinkhq/metascraper#344]
### Prerequisites

- [x ] I'm using the last version.
- [ ] My node version is the same as declared as `package.json`.

### Subject of the issue
Why requesting for node.js 14.0  if you don't use specifically a new feature from 14.0. 
We cannot deploy the last version on AWS lambda because of incompatible node.js version.
Could you downgrade back to 12.0 please.

### Steps to reproduce

deploy last version on AWS.

### Expected behaviour

It should build

### Actual behaviour

It won't build

**Comment by Kikobeats** [comment]
Hello,

The thing is I can only guarantee the library works as expected under Node 14 and 15 since they are the current LTS and the latest Node.js version available.

I can't guarantee the library works under old Node.js builds (like Node 12), but probably it will work.

What's the error npm is reporting you?

**Comment by rafaelrpinto** [comment]
Hi.

First of all thanks for the great project.

Same issue on google cloud functions. The latest engine is Node 12 and until node 14 is supported we cannot upgrade further than 5.16.6.

The issue is that when loading the function cloud functions run a `yarn install --production` which fails the engine validation for node 14. As of this time I could not find any way of customizing this install command to pass the ignore engine check.

So long story short, after 5.16.6 the library  started being unusable for cloud functions.

Since Node.js 12 will be maintained until April 2022 and the library do not require any of Node 14 features, in my opinion this restriction hurts more than benefits.

Thanks.

**Comment by Kikobeats** [comment]
can you try `yarn install --ignore-engines`?

**Comment by rafaelrpinto** [comment]
Actually the problem now seems worse because even fixing the version to 5.16.6 wont work as its dependencies require `@metascraper/helpers` with version `^5.16.6`:

```
metascraper-author@5.16.6:
  version "5.16.6"
  resolved "https://registry.yarnpkg.com/metascraper-author/-/metascraper-author-5.16.6.tgz#..."
...
  dependencies:
    "@metascraper/helpers" "^5.16.6"
```

Which ends up resolving to `5.16.15` which is incompatible with Node.js 12.

```
"@metascraper/helpers@^5.16.6":
  version "5.16.15"
  resolved "https://registry.yarnpkg.com/@metascraper/helpers/-/helpers-5.16.15.tgz#..."
 ...
```

**Comment by rafaelrpinto** [comment]
@Kikobeats as I mentioned when deploying the function google does not allow us to customize the install command. It will execute `yarn install --production` and I could not find a way of adding flags to that.

**Comment by Kikobeats** [comment]
restored Node 12 version at metascraper@5.16.16

can you confirm now is working fine with the latest version?

**Comment by rafaelrpinto** [comment]
@Kikobeats 
Yes. Now its deploying alright.
Thanks for the super fast response and taking the time to fix it.
